index.html

```diff
@@ -6,21 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <script src="https://kit.fontawesome.com/8c1870b6fa.js" crossorigin="anonymous"></script>   
     <link rel="stylesheet" href="style.css" />
+    <script src="./script.js"></script>
     <title>My Professional Portfolio</title>
   </head>
   <body>
     <header>
       <a href="./index.html"><img class="logo" src="./public/logo.png" alt="logo"> </a>
       <nav>
-        <div class="menu">
-          <svg width="750px" height="750px" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" fill="none">
-            <path stroke="#6070FF" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 8h22M5 16h22M5 24h22"/>
-          </svg>
+        <div class="menu" >
+          <button class="menu-button" onclick="toggleDropdown()"> <i class="fa fa-bars"></i></button> 
         </div>
+        <div class="dropdown">
+          <div id="dropdownContent" class="dropdown-content">
+          <span class="close" onclick="closeDropdown()">&times;</span>
+          <a href="./index.html" onclick="closeDropdown()">Portfolio</a>
+          <a href="#about-me" onclick="closeDropdown()">About</a>
+          <a href="#contact-me" onclick="closeDropdown()">Contact</a>
+        </div>
+    </div>
```

Comment:
- Good job done so far, but you have not styled your mobile menu according to the Figma template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
![ethaehre](https://github.com/Kaiserabbas/Responsive-Portfolio/assets/73607512/e2160625-9508-4af6-85ef-d71e12712d9e)

Design template:
![ryjsrtjsetj](https://github.com/Kaiserabbas/Responsive-Portfolio/assets/73607512/0384f714-0dcd-4796-b439-d2810ec930ce)

**OPTIONAL** - Please, it is not advisable to call Javascript functions in your HTML file. Ensure all DOM manipulations are done in the js file. Please fix this 🙏

Reply:
I have done the changes required to match figma design has been completed. Kindly approve.